src/lib/toastr/toast.component.ts

```diff
@@ -39,20 +39,17 @@ import { ToastrService } from './toastr.service';
   `,
   animations: [
     trigger('flyInOut', [
-      state(
-        'inactive',
-        style({
-          display: 'none',
-          opacity: 0
-        })
-      ),
-      state('active', style({})),
+      state('inactive', style({ opacity: 0 })),
+      state('active', style({ opacity: 1 })),
```

Comment:
Does this line also fix #602 & #605? From a quick test with bootstrap 4.3.1 cdn and the latest code (9.2.0) the toasts are showing up now without having to overwrite the css myself

Reply:
Wait hang on. I'm using the normal css. One moment...

Reply:
Guess it doesn't completely fix it. It's fixed when using the `ToastComponent`, but when using the `ToastNoAnimation` component the issue still exists.

Reply:
I think to fix it for `ToastNoAnimation` we'd want to apply something similar here:
https://github.com/scttcper/ngx-toastr/blob/720af1ac29e5709ed67f890ce722a625dfb321c6/src/lib/toastr/toast-noanimation.component.ts#L56-L62

Instead of binding to "style.display" we could bind to just "style" and return both the display and opacity?

Reply:
i don't know if you can return an object there